src/Saltoapis.Auth/SaltoapisClientCredentialsProvider.cs

```diff
@@ -92,15 +95,23 @@ class SaltoOAuthClient : OAuthClientCredentialsProvider
 
         SaltoTokenResponse token;
 
-        readonly String clientId;
-        readonly String clientSecret;
+        readonly String   clientId;
+        readonly String   clientSecret;
+        readonly String[] scopes;
+
+        public static readonly String[] DefaultScopes = new String[] {
+            "openid", "offline", "https://saltoapis.com/auth/nebula"
```

Comment:
@glerchundi when the user does not specify any scope we are requesting these by default, is this ok? Or should we force them to specify scopes they want?

Reply:
I would say that we can start without defining this default scopes.